www/js/lib/uiUtil.js

```diff
@@ -32,6 +32,38 @@ if (webpMachine) {
 }
 
 define(rqDef, function(settingsStore) {
+
+    /**
+     * Displays a Bootstrap alert or confirm dialog box depending on the options provided
+     * @param {*} node 
+     * @param {*} nodeAttribute 
+     * @param {*} content 
+     * @param {*} mimeType 
+     * @param {*} callback 
+     */
+    function systemAlert(label, message, isConfirm){
+        return new Promise(function(resolve, reject) {
+            document.getElementById("modalLabel").innerHTML = label;
+            document.getElementById("modalText").innerHTML = message;
+            if(isConfirm){
```

Comment:
Space after `if` and after close parenthesis on this line (it's our style, also conforms with ESLint).